Cargo.toml

```diff
@@ -155,6 +156,7 @@ edition = "2021"
 
 [workspace.dependencies]
 Inflector = "0.11.4"
+agave-transaction-view = { path = "transaction-view", version = "=2.1.0" }
```

Comment:
oh, do we suppose to call new crates `agave-*` instead of `solana-*` from now on?

Reply:
Seems more appropriate to me - type is primarly intended as an internal agave-validator type - not something that the solana community will use at large.

Reply:
https://discord.com/channels/428295358100013066/560503042458517505/1266067084069310606

Sounds like since this is targetted towards the validator code itself, `agave-X` is the naming convention we should be using.